tests/plugins_test.go

```diff
@@ -197,6 +203,67 @@ func TestBufPluginConfig(t *testing.T) {
 	}
 }
 
+func TestGoMinVersion(t *testing.T) {
```

Comment:
Sample failure from reverting the change earlier in the week to protocolbuffers/go v1.34.2:

```
=== NAME  TestGoMinVersion/protocolbuffers/go@v1.34.2
    plugins_test.go:236: 
        	Error Trace:	/Users/pkw/dev/plugins/tests/plugins_test.go:236
        	Error:      	Not equal: 
        	           	expected: "1.16"
        	           	actual  : "1.20"
        	           	
        	           	Diff:
        	           	--- Expected
        	           	+++ Actual
        	           	@@ -1 +1 @@
        	           	-1.16
        	           	+1.20
        	Test:       	TestGoMinVersion/protocolbuffers/go@v1.34.2
        	Messages:   	expected go plugin registry.go.min_version to be equal to the max version of its dependencies "1.16" (1.20)%!(EXTRA string=google.golang.org/protobuf@v1.34.2)
--- FAIL: TestGoMinVersion (0.39s)
    --- FAIL: TestGoMinVersion/protocolbuffers/go@v1.34.2 (0.04s)
```